spec/features/idv/doc_auth/hybrid_handoff_spec.rb

```diff
@@ -13,7 +13,6 @@
   let(:facial_match_required) { false }
   context 'split doc auth', allow_browser_log: true do
     before do
-      allow(IdentityConfig.store).to receive(:doc_auth_separate_pages_enabled).and_return(true)
```

Comment:
it looks like this context `split doc auth` is no longer needed as well ... it seems it was wrapped around our existing tests with the addition of this line you're removing 🤔